test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -11141,6 +11141,22 @@ BOOST_AUTO_TEST_CASE(swap_peephole_optimisation)
 	BOOST_CHECK(callContractFunction("div(uint256,uint256)", u256(0), u256(1)) == encodeArgs(u256(0)));
 }
 
+BOOST_AUTO_TEST_CASE(symmetric_literal_comparisons)
+{
+	char const* sourceCode = R"(
```

Comment:
Sorry, I only noticed this now, this should not be an EndToEndTest, since we check the syntax and not the semantics. Can you change this to a syntax test (in the new format)?

Reply:
Done